### PR TITLE
Add Dockerfile for aasp container

### PR DESCRIPTION
**Pull request by stevendongatmsft**

### CODE REVIEW

File: Dockerfile.aaa [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ARG GOLANG_VERSION=1.17
+
+FROM golang:${GOLANG_VERSION} AS build
+ENV PROJECT_DIR=/go/src/github.com
+WORKDIR ${PROJECT_DIR}
+RUN git clone https://github.com/microsoft/confidential-sidecar-containers.git 
+RUN git clone https://github.com/container-investigations/aaa.git
+RUN cd ${PROJECT_DIR}/confidential-sidecar-containers/tools/get-snp-report && make && mv bin/get-snp-report / && mv bin/get-fake-snp-report /
+RUN cd ${PROJECT_DIR}/aaa && CGO_ENABLED=0 GOOS=linux go build -o /aaa -ldflags="-s -w" github.com/container-investigations/aaa/cmd/aasp
```
Comment (jxyang) [inline Dockerfile.aaa]: Rename /aaa to /aasp

Comment (jxyang) [reply Dockerfile.aaa]: Please rename the file to Dockerfile.build, and put under `docker`.

Comment (stevendongatmsft) [reply Dockerfile.aaa]: all done

**Review by jxyang — APPROVED** [review]
Thanks for the contribution, @stevendongatmsft

**Pull request merged**